tests/PSDocs.Azure.Tests/template-test/metadata.json

```diff
@@ -0,0 +1,9 @@
+{
+    "$schema": "https://aka.ms/azure-quickstart-templates-metadata-schema#",
+    "itemDisplayName": "Storage Account",
```

Comment:
Perhaps this should be named slightly differently to show the effect of having metadata.json?

Reply:
Thanks Vic. It's a predefined schema so the properties are what they are. I'll expand the mapping table to provide some additional detail, which should hopefully address it.